src/gui/src/main.cpp

```diff
@@ -54,7 +54,7 @@ bool checkMacAssistiveDevices();
 
 void copy_qsettings(const QSettings &src, QSettings &dst)
 {
-    auto keys = src.allKeys();
+    const auto keys = src.allKeys();
```

Comment:
There's no need for `const` for non-reference types. I've yet to hit a bug that `const` would have prevented, however it introduces a lot of visual noise. Little benefit with certain costs.

Reply:
This is due to a clazy check for detachment on the ranged for. We can't use `qAsConst` with the type so we make it `const` prevents the issue.
You can read it more about it [here](https://github.com/KDE/clazy/blob/master/docs/checks/README-range-loop-detach.md).